sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/PendingTransaction.kt

```diff
@@ -31,7 +31,7 @@ sealed class TransactionRecipient {
         override fun toString() = "TransactionRecipient.Address"
     }
 
-    data class Account(val accountValue: cash.z.ecc.android.sdk.model.Account) : TransactionRecipient() {
+    data class Account(val accountZip32Index: Int) : TransactionRecipient() {
```

Comment:
This change is incorrect. The value currently returned from the SQL view is the internal account ID, not the ZIP 32 account index. (This is where the SDKs were conflating internal account IDs with ZIP 32 account indices previously; we should not persist with this conflation.)

Reply:
This one is tricky, as we currently don't have a way to get `Account` here. I just changed the parameter to `val accountId: Int` in this PR. And in #1640 I added this small change [d9cd693](https://github.com/Electric-Coin-Company/zcash-android-wallet-sdk/pull/1640/commits/d9cd693f8ac805628680e9c06dffa8b96cf61a6f). IIUC the iOS SDK has the same bug. I'll rise it.